frontend/lib/svg/letter-of-complaint.svg

```diff
@@ -0,0 +1 @@
2.4h-.26q.55,8.58.55,12.24v20.16h-8.11V334.72h11.45l18.61,32.08h.2q-.43-8.35-.44-11.81V334.72h8.18Z"/><path className="cls-4" d="M970.93,355.72q0,10.57-6,16.2t-17.38,5.63H935.4V334.72h13.44q10.5,0,16.29,5.53T970.93,355.72Zm-9.43.24q0-13.8-12.19-13.8h-4.83v27.89h3.9Q961.51,370.05,961.5,356Z"/><path className="cls-4" d="M979.81,377.55V334.72h9.08v35.33h17.38v7.5Z"/><path className="cls-4" d="M1052.58,356.07q0,10.64-5.27,16.35t-15.12,5.71q-9.84,0-15.11-5.71T1011.8,356q0-10.69,5.29-16.34t15.16-5.64q9.87,0,15.11,5.69T1052.58,356.07Zm-31.25,0q0,7.19,2.72,10.81t8.14,3.64q10.88,0,10.87-14.45t-10.81-14.47q-5.41,0-8.17,3.65T1021.33,356.07Z"/><path className="cls-4" d="M1070.54,361.11v16.44h-9.08V334.72h12.48q8.73,0,12.92,3.18t4.19,9.65a11.32,11.32,0,0,1-2.08,6.72,13.79,13.79,0,0,1-5.89,4.62q9.68,14.44,12.6,18.66H1085.6l-10.22-16.44Zm0-7.38h2.93q4.3,0,6.36-1.44a5.11,5.11,0,0,0,2.05-4.51,4.69,4.69,0,0,0-2.09-4.33q-2.1-1.29-6.49-1.29h-2.76Z"/><path className="cls-4" d="M1136.61,355.72q0,10.57-6,16.2t-17.39,5.63h-12.13V334.72h13.45q10.49,0,16.29,5.53T1136.61,355.72Zm-9.44.24q0-13.8-12.18-13.8h-4.84v27.89h3.9Q1127.18,370.05,1127.17,356Z"/><path className="cls-4" d="M1197.79,101.45a6,6,0,0,0-5.16,2.5,11.65,11.65,0,0,0-1.83,7q0,9.31,7,9.31a22.22,22.22,0,0,0,7.1-1.46v5a19.71,19.71,0,0,1-7.65,1.42q-6.08,0-9.29-3.68t-3.22-10.58a17.22,17.22,0,0,1,1.58-7.61,11.43,11.43,0,0,1,4.54-5,13.35,13.35,0,0,1,6.94-1.75,18.82,18.82,0,0,1,8.15,2l-1.9,4.8a33.48,33.48,0,0,0-3.14-1.29A9.36,9.36,0,0,0,1197.79,101.45Z"/><path className="cls-4" d="M1226.85,124.77h-16V96.93h16v4.84h-10.13v6.11h9.43v4.84h-9.43v7.18h10.13Z"/><path className="cls-4" d="M1238.56,114.09v10.68h-5.9V96.93h8.11q5.67,0,8.4,2.07t2.72,6.27a7.38,7.38,0,0,1-1.35,4.37,8.89,8.89,0,0,1-3.83,3q6.29,9.39,8.19,12.13h-6.55l-6.65-10.68Zm0-4.8h1.91a7.31,7.31,0,0,0,4.13-.93,3.32,3.32,0,0,0,1.33-2.93,3,3,0,0,0-1.36-2.82,8.24,8.24,0,0,0-4.22-.84h-1.79Z"/><path className="cls-4" d="M1269.15,124.77h-5.91V101.85h-7.56V96.93h21v4.92h-7.56Z"/><path className="cls-4" d="M1281,124.77V96.93h5.9v27.84Z"/><path className="cls-4" d="M1299.71,124.77h-5.81V96.93h16v4.84h-10.15V109h9.45v4.82h-9.45Z"/><path className="cls-4" d="M1315.31,124.77V96.93h5.9v27.84Z"/><path className="cls-4" d="M1344.25,124.77h-16V96.93h16v4.84h-10.13v6.11h9.43v4.84h-9.43v7.18h10.13Z"/><path className="cls-4" d="M1373.16,110.59q0,6.87-3.91,10.53t-11.31,3.65h-7.88V96.93h8.74q6.81,0,10.59,3.6T1373.16,110.59Zm-6.13.15q0-9-7.93-9H1356V119.9h2.54Q1367,119.9,1367,110.74Z"/><path className="cls-4" d="M1250,166.77l-6.7-21.84h-.18q.38,6.67.37,8.89v13h-5.28V138.93h8l6.59,21.29h.11l7-21.29h8v27.84h-5.5V153.6c0-.63,0-1.34,0-2.16s.11-3,.26-6.47h-.17l-7.18,21.8Z"/><path className="cls-4" d="M1292,166.77l-2-6.62h-10.15l-2,6.62h-6.36l9.83-28h7.21l9.87,28Zm-3.43-11.57q-2.8-9-3.15-10.19c-.24-.79-.41-1.41-.51-1.87-.42,1.63-1.62,5.64-3.6,12.06Z"/><path className="cls-4" d="M1301.88,166.77V138.93h5.9v27.84Z"/><path className="cls-4" d="M1314.79,166.77V138.93h5.91v23H1332v4.87Z"/></svg>
```

Comment:
Oof, yeah, it's kind of funky that this `.svg` file has a `className` attribute in it.  I guess we should file an issue to move this out into a standard 'ol static file, unless there are compelling reasons to keep it as an embedded SVG...